exported.txt

```diff
@@ -1,7 +1,5 @@
 {
     global:
-        _Z6renderPN3agl11DrawContextEPN4sead10TextWriterE;
-        __custom_init;
-        __custom_fini;
+        _Z21teleportMarioIfNeededv;
```

Comment:
This is the mangled name for our `teleportMarioIfNeeded` function; I grabbed it using `nm build310/main.o` (that build folder name is likely to change in an upcoming PR, but that's the root idea — use `nm` on `main`'s object file).